### ISSUE TITLE
probleme for contacts

### ISSUE THREAD

**Issue by cabbes** [polytan02/rainloop_ynh#1]
Hello,

I have installed this app to my yunohost instance, and I have 2 problems with contacts : 
- I had to create myself the rainloop base and a rainloop user in mysql
- I have to configure myself carddav sync with baikal

Is it possible to integrate automatically the carddav sync with baikal ? It's not a problem for me to configure this, but it's more difficult for the other members of my family.

Thanks !

**Comment by polytan02** [comment]
Hello!First, thank you for your message.This is a work in progress and also my first yunohost package.I configure the installation very easily to create a database for rainloop, but not one per user. I don't know what is the interest of the extra database as baikal should manage it, don't you think ?  I'm not sure how to set up the carddav connection automatically, but I agree that it's not very practical for each user to add the baikal address.Actually, when I tried it on my server, the contact created by rainloop was not added ‎to the baikal server, so I dropped this.Is it working on your side ? I do need to create an update script, but for now it doesn't keep the current configuration. If you are to update the app, it will simply delete everything and start from new.                                                                                                                                                                                                                                                                          Maxime                                                                                                                                                                                                                 De: cabbesEnvoyé: vendredi 6 février 2015 15:47À: polytan02/rainloop_ynhRépondre à: polytan02/rainloop_ynhObjet: [rainloop_ynh] probleme for contacts (#1)Hello,

I have installed this app to my yunohost instance, and I have 2 problems with contacts : 


I had to create myself the rainloop base and a rainloop user in mysql
I have to configure myself carddav sync with baikal


Is it possible to integrate automatically the carddav sync with baikal ? It's not a problem for me to configure this, but it's more difficult for the other members of my family.

Thanks !

—Reply to this email directly or view it on GitHub.
